lib/src/main/java/co/nimblehq/recentapps/thumbnailhiding/RecentAppsThumbnailHidingLifecycleTracker.kt

```diff
@@ -0,0 +1,70 @@
+package co.nimblehq.recentapps.thumbnailhiding
+
+import android.app.Activity
+import android.app.Application
+import android.os.Bundle
+
+class RecentAppsThumbnailHidingLifecycleTracker : Application.ActivityLifecycleCallbacks {
+
+    private var hardwareKeyWatcher: HardwareKeyWatcher? = null
+
+    override fun onActivityCreated(activity: Activity, savedInstanceState: Bundle?) {
+    }
+
+    override fun onActivityStarted(activity: Activity) {
+        if (hardwareKeyWatcher != null && hardwareKeyWatcher?.context != activity) {
+            hardwareKeyWatcher?.stopWatch()
+            hardwareKeyWatcher = null
+        }
+        hardwareKeyWatcher = HardwareKeyWatcher(activity).apply {
+            setOnHardwareKeysPressedListenerListener(object :
+                HardwareKeyWatcher.OnHardwareKeysPressedListener {
+                override fun onHomePressed() {
+                    activity.triggerRecentAppsMode(true)
+                }
+
+                override fun onRecentAppsPressed() {
+                    activity.triggerRecentAppsMode(true)
+                }
+            })
```

Comment:
Why don't we combine 2 callbacks? 🤔  As I can see, both cases would handle same action, or will we need to handle specific cases on some devices? 🤔

Reply:
@hoangnguyen92dn I would like to keep pure handling on `HardwareKeyWatcher` that will trigger home or recent key.

Reply:
👌